lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -268,7 +269,12 @@ module InternalConstants
           proc { |spec,| __FILE__.include?(spec.full_gem_path) }) do |spec,|
             spec.version.to_s
           end
-        matching_version
+        if matching_version
+          matching_version
+        else
+          # Return a valid but obviously wrong value.
+          '0.0.0-unknown'
+        end
```

Comment:
This whole block is just `matching_version || '0.0.0-unknown'`.

Reply:
Thanks. I prefer the current version, if that's OK with you.

Reply:
Unfortunately, the Ruby community prefers the other version: https://github.com/rubocop-hq/ruby-style-guide#if-as-a-modifier. It's also consistent with, e.g., [line 790](https://github.com/GoogleCloudPlatform/fluent-plugin-google-cloud/pull/356/files#diff-0d6486c76b449a4c9242958738de9cdbR790) below...